### PR TITLE
Remove install_istio flag

### PR DESCRIPTION
**Pull request by smshareef**
Commented out code for disabling the install_istio flag. Functionality is disabled to remove dependency from istio.io/istio package. Vulnerabilties were detected by quay.io security scan of aci-containers-controller and aci-containers-operator images for istio.io/istio package

### CODE REVIEW

File: provision/acc_provision/acc_provision.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def prepare_nadvlanmap(file_path):
                 all_resources[resource_header].append(resource_item)
     except Exception as ex:
         print("Error while preparing yaml contents from given CSV %s"
-            " file. Error: %s" % (file_path, ex))
+              " file. Error: %s" % (file_path, ex))
```
Comment (akhilamohanan) [inline provision/acc_provision/acc_provision.py:152]: is this change required?

**Comment by snaiksat** [comment]
Thanks @smshareef . The commit message is a bit misleading. The install_istio config is already disabled by default. What we are taking away is the ability to enable it.

Disabling the configuration to enable install_istio to remove dependency from istio.io/istio package since this feature will not be supported going forward. For the past few cycles we have moved towards supporting Red Hat Service Mesh operator.

File: provision/acc_provision/templates/provision-config.yaml [hunk 2]
```diff
@@ -157,7 +157,6 @@ registry:
     #enable_updates: True # Default is False
 
 #istio_config:
-  # install_istio: False         # default is True
   # install_profile: "default"   # override if needed, default is "demo"
```
Comment (snaiksat) [inline provision/acc_provision/templates/provision-config.yaml:160]: this whole section needs to be removed

**Review by snaiksat — CHANGES_REQUESTED** [review]

File: provision/acc_provision/templates/provision-config.yaml (hunk 2 above)
Comment (smshareef) [reply provision/acc_provision/templates/provision-config.yaml]: Done, thanks for the review @snaiksat.

**Review by akhilamohanan — APPROVED** [review]

**Pull request merged**